### DISCUSSION TITLE
Нубский вопрос №3

### DISCUSSION THREAD

**Discussion by KainXVIII** [Anonym-tsk/nfqws-keenetic#97] [Q&A]
В шаге 
> Вместе с этим рекомендуется [настроить использование DoT/DoH](https://help.keenetic.com/hc/ru/articles/360007687159).

нужно ли вводить все эти адреса из статьи и достаточно только установить компоненты DoT/DoH?

**Comment by pwgms213** [comment]
Можете сделать по этой инструкции, получите сразу антизапрет и DOH
https://ntc.party/t/%D0%BE%D0%B1%D1%85%D0%BE%D0%B4-%D0%B1%D0%BB%D0%BE%D0%BA%D0%B8%D1%80%D0%BE%D0%B2%D0%BE%D0%BA-%D0%B2-%D1%80%D0%BE%D1%81%D1%81%D0%B8%D0%B8-%D1%87%D0%B5%D1%80%D0%B5%D0%B7-%D0%B0%D0%BD%D1%82%D0%B8%D0%B7%D0%B0%D0%BF%D1%80%D0%B5%D1%82-%D0%BD%D0%B0-%D0%BF%D1%80%D0%BE%D1%88%D0%B8%D0%B2%D0%BA%D0%B5-keenetic/7768

**Reply by KainXVIII** [reply]
> Можете сделать по этой инструкции, получите сразу антизапрет и DOH https://ntc.party/t/%D0%BE%D0%B1%D1%85%D0%BE%D0%B4-%D0%B1%D0%BB%D0%BE%D0%BA%D0%B8%D1%80%D0%BE%D0%B2%D0%BE%D0%BA-%D0%B2-%D1%80%D0%BE%D1%81%D1%81%D0%B8%D0%B8-%D1%87%D0%B5%D1%80%D0%B5%D0%B7-%D0%B0%D0%BD%D1%82%D0%B8%D0%B7%D0%B0%D0%BF%D1%80%D0%B5%D1%82-%D0%BD%D0%B0-%D0%BF%D1%80%D0%BE%D1%88%D0%B8%D0%B2%D0%BA%D0%B5-keenetic/7768

Да я уже эту байду с entware+nfqws установил на флешку роутера, надо бы хоть попробовать. Что-то открывается (x.com), а что-то типа дискорда и флибусты всё равно нет, ютуб на телевизоре тоже не пашет (на пк нормально)
PS - discord  заработал, просто тут не сразу определяется, что заблокировано..

**Comment by KainXVIII** [comment]
Но вопрос из первого поста остаётся в силе.

**Comment by Anonym-tsk** [comment]
Нужно вводить адреса из статьи
